bootstrap.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
```

Comment:
Was there a reason you needed to remove this line?

Reply:
It was my mistake. When I was installing the development tools I had several issues. I've thought that wrong python installation was used.